src/components/Header/HeaderAlert.tsx

```diff
@@ -45,6 +45,9 @@ const HeaderAlert = ({
   const onClose = () => {
     onDismiss && onDismiss();
     setAlertVisible(false);
+    if (timer) {
+      clearTimeout(timer);
```

Comment:
This doesn't seem related to any of the preview changes, but nonetheless a good change 👍